k8s/gcp-pubsub/turbinia-output-filestore.yaml

```diff
@@ -0,0 +1,12 @@
+apiVersion: v1
+kind: PersistentVolume
+metadata:
+  name: output
+spec:
+  capacity:
+    storage: 1T
+  accessModes:
+  - ReadWriteMany
+  nfs:
+    path: /output
```

Comment:
What is this path for?  Is this supposed to be `/mnt/output`?

Reply:
yep, specifying a path is a required attribute when setting up GCP Filestore

Reply:
Do we need to update this to be `/mnt/output` instead of just `/output` then?

Reply:
No since that's the path of the nfs specified when creating the Filestore instance, `/mnt/output` is the path when it gets mounted to container itself

Reply:
Got it, thanks.